<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Sebastian Pipping</author>
 <class>toSearchReplaceUI</class>
 <widget class="QWidget" name="toSearchReplaceUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>656</width>
    <height>78</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>F&amp;ind:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>SearchText</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="SearchPrevious">
     <property name="toolTip">
      <string>Jump to previous match</string>
     </property>
     <property name="icon">
      <iconset resource="../tora.qrc">
       <normaloff>:/icons/find_prev.png</normaloff>:/icons/find_prev.png</iconset>
     </property>
     <property name="shortcut">
      <string>Ctrl+P</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="SearchText">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="toolTip">
      <string>Text to search for</string>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="maxCount">
      <number>30</number>
     </property>
     <property name="insertPolicy">
      <enum>QComboBox::InsertAtTop</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="4" colspan="2">
    <widget class="QComboBox" name="SearchMode">
     <property name="toolTip">
      <string>Search mode</string>
     </property>
     <property name="maxVisibleItems">
      <number>4</number>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>Plain Text</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Regular Expression</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="SearchNext">
     <property name="toolTip">
      <string>Jump to next match</string>
     </property>
     <property name="icon">
      <iconset resource="../tora.qrc">
       <normaloff>:/icons/find_next.png</normaloff>:/icons/find_next.png</iconset>
     </property>
     <property name="shortcut">
      <string>Ctrl+N</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="ReplacementText">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Text to replace with</string>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="Replace">
     <property name="toolTip">
      <string>Replace next match</string>
     </property>
     <property name="icon">
      <iconset resource="../tora.qrc">
       <normaloff>:/icons/replace_next.png</normaloff>:/icons/replace_next.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Rep&amp;lace:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>ReplacementText</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="ReplaceAll">
     <property name="toolTip">
      <string>Replace all matches</string>
     </property>
     <property name="icon">
      <iconset resource="../tora.qrc">
       <normaloff>:/icons/replace_all.png</normaloff>:/icons/replace_all.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QCheckBox" name="WholeWords">
     <property name="text">
      <string>Whole words</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QCheckBox" name="MatchCase">
     <property name="toolTip">
      <string>Case-sensitive searching</string>
     </property>
     <property name="text">
      <string>&amp;Match case</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>SearchText</tabstop>
  <tabstop>SearchNext</tabstop>
  <tabstop>SearchPrevious</tabstop>
  <tabstop>ReplacementText</tabstop>
  <tabstop>Replace</tabstop>
  <tabstop>ReplaceAll</tabstop>
  <tabstop>SearchMode</tabstop>
  <tabstop>MatchCase</tabstop>
 </tabstops>
 <resources>
  <include location="../tora.qrc"/>
 </resources>
 <connections/>
</ui>
